### PR TITLE
Auto-format buildSrc

### PR DESCRIPTION
**Pull request by pugnascotia**
Opt-in buildSrc for automatic formatting. This required a config tweak
in order to pick up all the Java sources, and as a result more files are
now found in the Enrich plugin, that were previously missed.

@martijnvg would you mind reviewing the handful of changed files for Enrich?

### CODE REVIEW

File: buildSrc/src/main/java/org/elasticsearch/gradle/Jdk.java [hunk 1]
```diff
@@ -135,7 +135,8 @@ void finalizeValues() {
         }
         version.finalizeValue();
         platform.finalizeValue();
-        vendor.finalizeValue();;
+        vendor.finalizeValue();
+        ;
```
Comment (martijnvg) [inline buildSrc/src/main/java/org/elasticsearch/gradle/Jdk.java]: maybe just remove this empty statement?

**Review by martijnvg — APPROVED** [review]
Enrich wise LGTM

**Review by mark-vieira — APPROVED** [review]
This all looks pretty reasonable. 👍

**Comment by pugnascotia** [comment]
@martijnvg thank you, good spot, I've removed the semicolon.

@mark-vieira I moved the two Java files out of the Groovy directory, but I've left the Spotless config as-is because it found some missed-out Java files in the Enrich projet as well. I've also replaced a few String concatentations with `format()` calls, what do you think?

**Review by mark-vieira — APPROVED** [review]
LGTM 👍

**Comment by pugnascotia** [comment]
@elasticmachine run elasticsearch-ci/default-distro

**Pull request merged**